### DISCUSSION TITLE
Why are "Issues" not closed?

### DISCUSSION THREAD

**Discussion by v-Kaefer** [ChrisTitusTech/winutil#1538] [General]
Most Issues, even when solved, end up not being closed and when they are, it takes a really long time for just a couple of them to get closed. Is there someone i can tag, that is active and able to close issues? Or is @ChrisTitusTech the only one able to?

**Comment by v-Kaefer** [comment]
This was the **WORST** time to activate watch on issues, but i'm so glad you added a bot for it @ChrisTitusTech.

**Reply by ChrisTitusTech** [reply]
Yeah, saw these getting away from me and automatically closing inactive issues was the best solution.

I'm currently in a heavy refactor and simplifying the overall script to increase its speed.
